libs/shared/providers/src/notifications/components/NotificationsProvider.tsx

```diff
@@ -28,7 +28,7 @@ const NotificationsWrapped = ({ children, ...rest }: StackProps) => {
           {...rest}
           sx={{
             position: 'fixed',
-            top: (theme) => theme.spacing(8),
+            top: (theme) => theme.spacing(16), // Adjusted lower to look good while simplify OETH banner exists.
```

Comment:
this will impact all dApps using this Provider while this update should only impact oeth. 
You can override the defaults by using the `containerProps` attribute in `apps/oeth/src/main.tsx`
```
[
        NotificationsProvider,
        {
          containerProps: { sx: { top: (theme: Theme) => theme.spacing(16) } },
        },
      ],
```

Reply:
woops -- thanks for catching